WordPress/Classes/ViewRelated/Themes/ThemeBrowserViewController.swift

```diff
@@ -780,6 +793,12 @@ public protocol ThemePresenter: class {
                 let successMessage = String(format: successFormat, theme?.name ?? "", theme?.author ?? "")
                 let manageTitle = NSLocalizedString("Manage site", comment: "Return to blog screen action when theme activation succeeds")
                 let okTitle = NSLocalizedString("OK", comment: "Alert dismissal title")
+
+                self?.activityIndicator.stopAnimating()
+                self?.activateButton?.customView = nil
+                self?.activateButton?.isEnabled = false
+                self?.activateButton?.title = ThemeAction.active.title
```

Comment:
nit: totally not a big deal, but maybe we can group these calls in an helper method for readability? Same with the two calls to start the animation above.

Reply:
Done in 3a57070fa24d7e0913ae182564a7da9e800d56fb